### PR TITLE
Web Api ( capstone Module 2) 

### PR DESCRIPTION
**Pull request by andym80**
In this project, we must choose an external API and develop a page with comments, pop-up with a form, and a likes button. We are using Javascript with ES6, HTML, and CSS. Webpack and Jest. 
My task was centered on the Homepage to develop this web, getting the information from an API, and creating the Likes button functionality. 
some of the Tasks : 

- Setup the project with Webpack and Jest 
- Find an External API 
- Create a feature:  add a new Like. 
- Display the number of Likes for each item on the Homepage. 
- Add the Items counter on the Homepage
- Test 

Any suggestions are welcome! Thank you  

@SlowKingV 
@andym80

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @andym80  and  @SlowKingV , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] Those not have a [Readme](https://github.com/microverseinc/readme-template) file, please try to include it before submitting it for the next review.
- [ ] Please your partner should participate in the project by contributing to the project repo **Make commits**
- [ ] Do not forget your video presentation 
- [ ] Remember to use [GitHub](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md) flow

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.js [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+
+const getShowsInfo = async () => {
+  shows.forEach(async (show) => {
+    const showResponse = await fetch(`${API_URL}/shows/${show}`);// show information
+    const showData = await showResponse.json();
+    showsInfo.push({ id: showData.id, name: showData.name, image: showData.image.medium });
+  });
+};
+getShowsInfo().then(() => {
+});
```
Comment (SlowKingV) [inline src/index.js]: Kindly consider removing any empty function you have as it just uses unnecessary space in your file.

File: src/index.js [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+
+showsCounter();
```
Comment (SlowKingV) [inline src/index.js]: Your function `showsCounter()` seems to need an array as an argument. You might want to send an array that holds your shows like `showsInfo`

File: src/modules/greatAPI.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { getLikes, sendLikes } from './getLikes.js';
+
+const hintLikes = (array) => {
+  array.forEach((element) => {
+    element.addEventListener('click', () => {
+      sendLikes(element.id);
+      getLikes(element.id, element);
```
Comment (SlowKingV) [inline src/modules/greatAPI.js]: [OPTIONAL] Your functions `sendLikes()` and `getLikes()` are running in a synchronous way here and therefore the like counter doesn't update correctly. Consider making this an `async` function and `await` for both functions to finish.

File: src/modules/getLikes.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+const sendLikes = (name) => {
+  fetch(LikeAPI, {
+    method: 'POST',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      item_id: name,
+    }),
+  });
+};
```
Comment (SlowKingV) [inline src/modules/getLikes.js]: [OPTIONAL] As I mentioned in the previous comment, this function should be asynchronous like the other one so they wait for each other before completing.

**Review by SlowKingV — CHANGES_REQUESTED** [review]
# Changes Required
Your project is looking great so far!! 🎉 
However, there are still a few suggested changes to make it look even better!

File: src/index.js (hunk 1 above)
Comment (andym80) [reply src/index.js]: solved

**Review by SlowKingV — APPROVED** [review]

**Pull request merged**